crates/driver/src/infra/simulator/mod.rs

```diff
@@ -152,6 +152,8 @@ pub enum SimulatorError {
     Blockchain(#[from] blockchain::Error),
     #[error("enso error: {0:?}")]
     Enso(#[from] enso::Error),
+    #[error("the simulated gas {0:?} exceeded the gas limit {1:}")]
```

Comment:
```suggestion
    #[error("the simulated gas {0:?} exceeded the gas limit {1:?}")]
```

Reply:
Good catch! I removed the `:?` since `eth::Gas` implements `Display` now :ok_hand: